cypress/support/commands.js

```diff
@@ -43,11 +43,12 @@ Cypress.Commands.add('logout', () => {
     cy.wait('@logout');
 });
 
-Cypress.Commands.add('createRoom', () => {
+Cypress.Commands.add('createRoom', numRounds => {
     cy.route({
         method: 'POST',
         url: '/api/room/create'
     }).as('createRoom');
+    cy.get('[data-cy="numRounds-options"]').select(numRounds.toString());
```

Comment:
Sorry one more thing, for `data-cy`s can we have hyphenating instead of camel case e.g. `num-rounds-options`